hivemind/p2p/p2p_daemon_bindings/control.py

```diff
@@ -179,3 +282,9 @@ async def stream_handler(self, proto: str, handler_cb: StreamHandler) -> None:
 
         # if success, add the handler to the dict
         self.handlers[proto] = handler_cb
+
+
+class RemoteException(Exception):
```

Comment:
We already have `P2PHandlerError` for protobuf handlers, let's keep one of them.